controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -164,6 +164,12 @@ func (r *DataScienceClusterReconciler) Reconcile(ctx context.Context, req ctrl.R
 		}
 	}
 
+	// reconcile TrustyAI component
```

Comment:
This TrustyAI component sub-reconciliation is not needed, since all the sub-components are already reconciled above.

Reply:
ah, i must missed that fix PR, will add it here, thanks!